meetings/2022-01-26.md

```diff
@@ -0,0 +1,97 @@
+
+
+## Present
+* Matthieu Bosquet
+* Elf Pavlik
+* Wouter Termont
+* Tom Haegemans
+* Barath
+
+---
+
+## Announcements
+
+### Meeting Recordings and Transcripts
+* No audio or video recording, or automated transcripts without consent. Meetings are transcribed and made public. If consent is withheld by anyone, recording/retention must not occur.
+* Use panel chat and repository. Queue in call to talk.
+
+
+### Participation and Code of Conduct
+* [Join the W3C Solid Community Group](https://www.w3.org/community/solid/join), [W3C Account Request](http://www.w3.org/accounts/request), [W3C Community Contributor License Agreement](https://www.w3.org/community/about/agreements/cla/)
+* [Solid Code of Conduct](https://github.com/solid/process/blob/master/code-of-conduct.md), [Positive Work Environment at W3C: Code of Ethics and Professional Conduct](https://www.w3.org/Consortium/cepc/)
+* If this is your first time, welcome! please introduce yourself.
+
+
+### Scribes
+* Matthieu Bosquet
+* Wouter Termont
+
+
+### Introductions
+* Tom Haegemans: One of the co-founders of Digita and would like get more involved in the specification process
+
+---
+
+## Topics
+
+* Policies enforceable only by law https://github.com/solid/authorization-panel/issues/286
+*
+
+### Policies enforceable only by law
+
+* Matthieu: I think it is out of scope for WAC and ACP.
+* Elf: I think I agree, WAC and ACP would only cover the path that is enforced programatically. But maybe Data grants for example could cover some of it.
+* Matthieu: You're proposing that Data grants could be a good place to express not programatically enforceable permissions?
+* ELf: Yes, something at least available to the grantee and understand how they would be allowed to delegate their access. We need to understand what different authorization systems can do and make it user friendly to understand permissions and I don't think users should interact with ACLs (WAC or ACP). We should be careful that people are not mislead and don't mistakenly overpermission.
+* Tom: I agree that it's good to have a nice user experience, but also, giving feedback to the user is not necessarily sufficient, it doesn't address the problem of malicious apps. And what I see as very useful would be some sort of report on how one's data has been accessed.
+* Elf: Yes, when we have some kind of policy, we should indeed have some kind of report/timeline (as feedback).
+* Tom: Yes, I've been looking into data protection research and for example anna riselman? step away from consent and analyse what happens after a user has given consent. Consent can only go that far because the user needs to be able to understand what they're consenting to (let's say for example in the terms of use everyone just clicks ok). It is a real problem. By the way, I also shared a blogpost that I wrote on the subject a few years ago (https://www.digita.ai/news/2020/2/25/why-consent-is-flawed-and-what-organisations-can-do-about-it)
```

Comment:
```suggestion
* Tom: Yes, I've been looking into data protection research; for example, Anna Riselman(?)... Step away from consent, and analyse what happens after a user has given consent. Consent can only go so far because the user needs to be able to understand what they're consenting to (let's say, for example, in the terms of use, everyone just clicks ok). It is a real problem. By the way, I also shared [a blogpost that I wrote on the subject a few years ago](https://www.digita.ai/news/2020/2/25/why-consent-is-flawed-and-what-organisations-can-do-about-it)
```